tests/integration_tests/test_server.py

```
@@ -721,7 +721,7 @@ async def test_game_host_name_non_ascii(lobby_server):
         "command": "game_host",
         "mod": "",
         "visibility": "public",
-        "title": "ÇÒÖL GÃMÊ"
+        "title": "Ã‡Ã’Ã–L GÃƒMÃŠ"
```

Ruff (0.15.13)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)